packages/vats/src/types.js

```diff
@@ -38,11 +38,17 @@
  * @property {(key: string) => void} reserve Mark a key as reserved; will
  * return a promise that is fulfilled when the key is updated (or rejected when
  * deleted).
+ * @property {( key: string, newValue: unknown, newAdmin?: unknown) =>
+ *   Promise<any> } default Update if not already updated.  Return
```

Comment:
Can we assume the new value is the same type as the prior value?
```suggestion
 * @property {<T>( key: string, newValue: T, newAdmin?: unknown) =>
 *   Promise<T> } default Update if not already updated.  Return
```

Reply:
No, we can't make that assumption.